common/plugins/eu.esdihumboldt.hale.common.schema/src/eu/esdihumboldt/hale/common/schema/model/constraint/property/Reference.java

```diff
@@ -122,4 +122,15 @@ public Object extractId(Object refValue) {
 		return refValue;
 	}
 
+	/**
+	 * Converts the identifier of a referenced object to the reference value.
+	 * The default implementation just returns the given reference.
+	 * 
+	 * @param reference the reference
+	 * @return the reference value
+	 */
+	public Object toId(Object reference) {
```

Comment:
I think the method name is a bit misleading. It should convert an ID to a reference. (Opposed to the existing extractId method that converts a reference to an ID)